frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -2008,5 +2008,71 @@
     "flags": {
       "required": ["KNATIVE_EVENTING"]
     }
+  },
+  {
+    "type": "console.topology/data/factory",
+    "properties": {
+      "id": "knative-serving-topology-model-factory",
+      "priority": 100,
+      "resources": {
+        "$codeRef": "getKnativeResources.getKnativeServingResources"
+      },
```

Comment:
resources expects a `WatchK8sResourcesGeneric` and not a codeRef

Reply:
also since getKnativeServingResources always returns the same object we can embed the contents of that directly into the json file--no codeRef needed